client/src/components/DataFiles/DataFilesProjectMembers/DataFilesProjectMembers.jsx

```diff
@@ -85,10 +98,30 @@ const DataFilesProjectMembers = ({
   const columns = [
     memberColumn,
     {
-      Header: 'Access',
-      accessor: 'access',
+      Header: 'Permissions',
+      accessor: 'user.username',
+      id: 'project-role',
+      className: 'project-members__cell',
+      Cell: projectId
+        ? (el) => (
+            <ProjectRoleSelector projectId={projectId} username={el.value} />
+          )
+        : (el) => (
+            <span>{mapAccessToRoles(el.row.original.access).projectRole}</span>
+          ),
+    },
+    {
+      Header: 'Role',
+      accessor: 'user.username',
+      id: 'role',
       className: 'project-members__cell',
-      Cell: (el) => <span className={styles.access}>{el.value}</span>,
+      Cell: projectId
+        ? (el) => (
+            <SystemRoleSelector projectId={projectId} username={el.value} />
```

Comment:
Looks like system "viewers" aren't able to see other roles, so it 500s and renders "Error". I think non-admins should not only have selectors disabled, but have a different, simplified view-only modal that lists fellow project members and their project "permissions". 

![Screen Shot 2022-03-14 at 2 19 19 PM](https://user-images.githubusercontent.com/20326896/158245711-b0d08cec-ed35-457e-a7d4-591b025bd8bc.png)

Reply:
Can the simplified modal just be the same model with conditional rendering of elements? E.g.
* no "Add Member nor [Add] nor search
* no actions in the (unnamed) action column
* (if I understand correctly) no "Role" column